plat/common/x86/cpu_native.c

```diff
@@ -55,6 +55,13 @@ unsigned long read_cr2(void)
 /* This corresponds to an 85 (42 << 1 | 1) return value from QEMU */
 #define QEMU_ISA_DEBUG_EXIT_CRASH	42
 
+/* The port used by QEMU by default for the pvpanic device */
+#define QEMU_PVPANIC_EXIT_PORT		0x505
+/* This corresponds to a GUEST_PANICKED (1 << 0) event for QEMU */
+#define QEMU_PVPANIC_GUEST_PANICKED	1
+/* This corresponds to a GUEST_CRASHLOADED (1 << 1) event for QEMU */
```

Comment:
Is it defined as `1 << 1`? Then why not use that definition directly?

Reply:
Also would be nice to have a reference to the QEMU documentation of the pvpanic device in the comments.